src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -26,16 +25,17 @@ export const NavSearchResult: React.FC<NavSearchResultProps> = ({
     [image]
   );
 
+  const imageClasses = useMemo(
+    () =>
+      classNames("NavSearchResult__image", {
+        NavSearchResult__avatar: isAvatar,
+      }),
+    [isAvatar]
+  );
+
   return (
     <div className="NavSearchResult font-size--small" onClick={onClick}>
-      {isAvatar ? (
-        <UserAvatar
-          avatarSrc={image}
-          containerClassName="NavSearchResult__avatar"
-        />
-      ) : (
-        <div className="NavSearchResult__image" style={imageStyles} />
-      )}
+      <div className={imageClasses} style={imageStyles} />
```

Comment:
@sunny-viktoryia Why did we remove `UserAvatar` here out of curiosity?

Reply:
`NavSearchResult` component consists of `icon`, `title` and `description`. 

After `UserAvatar` was refactored, seems I cannot pass the icon there anymore, because I need user's object there. So I decided not to complicate `NavSearchResult` checking whethere a user is passed or an image. So now I just to change border-radius if it's an avatar.

But if you think that we should be consistent here as well. I can update the component in my next PR.

Reply:
My personal thoughts are that we should be ensuring our reusable components can be reused, rather than spreading similar'ish logic throughout the codebase. So my preference would be to figure a way for it to work with `UserAvatar`.

Thoughts @mike-lvov?

Reply:
Yes, it makes sense given that `UserAvatar` handles a few edge cases including anonMode & no profile pic set

Reply:
`UserAvatar` is back: https://github.com/sparkletown/sparkle/pull/1349